integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -973,6 +911,11 @@ public void verifyManagedServersRestarted() throws Exception {
     }
   }
 
+  /**
```

Comment:
please improve the javadoc.  it currently does not really add any information above and beyond the method signature

Reply:
Done